exercise_08/formatStringByWordsLength.js

```diff
@@ -16,3 +16,20 @@
  * CASE #2 * input: "This is short one."
  * output: "this is short one."
  */
+
+/**
+ * returns the parameter sentence in upper case if the number of sentence words if greater then 5, otherwise return lower case string 
+ * @param {string} sentence 
+ * @returns {string}
+ */
+const formatStringByWordsLength = function (sentence) {
+    const totalWords = sentence.split(' ');
+
+    if (totalWords.length > 5) {
```

Comment:
Avoid magic numbers. 

```suggestion
    if (totalWords.length > THRESHOLD) {
```

- [ ] Extract number to constant